src/css/Header.scss

```diff
@@ -1,24 +1,13 @@
 @import url('https://fonts.googleapis.com/css?family=Lobster');
 @import '_variables.scss';
 
-<<<<<<< HEAD
-.header {
```

Comment:
@hlhartley here as well. I think we really just want to remove line 4 and line 13?